api/src/main/java/com/strategyobject/substrateclient/api/pallet/democracy/Democracy.java

```diff
@@ -0,0 +1,271 @@
+package com.strategyobject.substrateclient.api.pallet.democracy;
+
+import com.strategyobject.substrateclient.pallet.annotation.*;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+@Pallet("Democracy")
+public interface Democracy {
```

Comment:
It looks like these are the errors for the democracy pallet, and not the events. 

Check here for reference: https://paritytech.github.io/substrate/master/pallet_democracy/pallet/enum.Event.html